scripts/test-captplanet.sh

```diff
@@ -89,3 +89,4 @@ rm small-upload-testfile
 rm big-upload-testfile
 rm multipart-upload-testfile
 rm big-download-testfile-ipv6
+rm multipart-upload-testfile
```

Comment:
There should be 7 things here (see above what files are created). Alternatively rename files such that they can all be removed together with `rm *-testfile`. 

Also these files should be created into $TMPDIR directory.

Reply:
This was a WIP PR but this has been addressed now 👍